ownCloud/AppDelegate.swift

```diff
@@ -134,8 +134,6 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 		OCExtensionManager.shared.addExtension(BackgroundFetchUpdateTaskAction.taskExtension)
 		OCExtensionManager.shared.addExtension(InstantMediaUploadTaskExtension.taskExtension)
 		OCExtensionManager.shared.addExtension(PendingMediaUploadTaskExtension.taskExtension)
-		OCExtensionManager.shared.addExtension(ImportPasteboardAction.actionExtension)
-		OCExtensionManager.shared.addExtension(CutAction.actionExtension)
```

Comment:
Why are these extensions removed? Looks like a merge conflict!?

Reply:
I moved these from the "Task extension" section of the code up to the section where the other action extensions are added (now in line 115 + 116).